misk-cron/src/main/kotlin/misk/cron/CronModule.kt

```diff
@@ -20,12 +21,12 @@ class CronModule(
 ) : KAbstractModule() {
   override fun configure() {
     install(FakeCronModule(zoneId, threadPoolSize, dependencies))
-    install(ServiceModule<RepeatedTaskQueue>(ForMiskCron::class))
+    install(ServiceModule<RepeatedTaskQueue>(ForMiskCron::class).dependsOn<ReadyService>())
```

Comment:
This is backwards incompatible -- Need to update any users of this who specify dependencies that ReadyService enhances